README.md

```diff
@@ -3,30 +3,88 @@ Styringsværktøj til IT projekter
 
 ## Setup locally
 
+### Preset
+Make sure you have a set of JSON files for testing import Commands.
+
 ### Start Docker containers
 
 ```sh
-docker-compose up -d
-docker-compose exec phpfpm composer install
-docker-compose exec phpfpm bin/console doctrine:migrations:migrate --no-interaction
+docker compose up -d
+docker compose exec phpfpm composer install
+docker compose exec phpfpm bin/console doctrine:migrations:migrate --no-interaction
 ```
 
 ### Create a super admin user
 
 ```sh
-docker-compose exec phpfpm bin/console fos:user:create --super-admin
+docker compose exec phpfpm bin/console fos:user:create --super-admin
 ```
 
 ### Access the site
 
 You should now be able to browse to the application
 
 ```
-open http://$(docker-compose port nginx 80)
+
+open "http://$(docker-compose port nginx 8080)"
+
 ```
 
 ## Import systems and reports
 ```sh
-bin/console itstyr:import:system PATH
-bin/console itstyr:import:report PATH
+docker compose exec phpfpm bin/console itstyr:import:system PATH
+docker compose exec phpfpm bin/console itstyr:import:report PATH
+```
+
+### Flowchart
```

Comment:
Why don't you use https://mermaid.js.org/syntax/entityRelationshipDiagram.html for showing entities and relations?